spec/lib/danger/ci_sources/support/pull_request_finder_spec.rb

```diff
@@ -128,6 +129,25 @@ def open_pull_requests_info
         finder(pull_request_id: "42", remote: true, logs: "not important").call
       end
     end
+
+    context "with bearer_token" do
+      before { ENV["DANGER_GITHUB_BEARER_TOKEN"] = 'hi' }
+      it "creates clients with bearer_token" do
+        allow(Octokit::Client).to receive(:new).with(
+          bearer_token: ENV["DANGER_GITHUB_BEARER_TOKEN"],
+          api_endpoint: be_a(String)
+        ) { spy("Octokit::Client") }
+
+        finder(pull_request_id: "42", remote: true, logs: "not important").call
+      end
+    end
+
+    after do
```

Comment:
these specs were fragile. These specs seem to share `ENV`s between test cases.
So they should be reset.